src/genericComponents/Button.tsx

```diff
@@ -7,27 +7,35 @@ const Button = (props: any) => {
   const { onClick, text, variant, ...rest } = props;
 
   return (
-    <CAGButton
-      className={`${variant}-class`}
-      onClick={onClick}
-      variant={variant}
-      {...rest}
-    >
-      {text}
-    </CAGButton>
+    <>
+      <CAGButton
+        className={`${variant}-class`}
+        onClick={onClick}
+        variant={variant}
+        {...rest}
+      >
+        {text}
+      </CAGButton>
+    </>
   );
 };
 
 const CAGButton = styled(BSButton)`
   border-radius: 25px;
   box-shadow: 2px 2px 20px rgba(0, 0, 0, 0.29);
   font-family: ${fonts.montserrat};
-  font-size: 12px;
-  font-weight: bold;
-  letter-spacing: 70;
-  line-spacing: 15;
-  padding: 16px 24px;
   text-transform: uppercase;
+  box-shadow: 0px 0px 8px 4px #0000000d;
```

Comment:
Note the box shadow property already on this component

Reply:
Missed that.